blockly/blocks/math.js

```diff
@@ -282,6 +282,35 @@ Blockly.Blocks['library_math_round'] = {
     onchange: Blockly.Blocks.requireInFunction
 };
 
+Blockly.Blocks['library_math_assignment_by'] = {
+    /**
+     * Block for rounding functions.
+     * @this Blockly.Block
+     */
+    init: function() {
+        var OPERATORS =
+            [[Blockly.Msg.MATH_ASSIGNMENT_BY_PLUS_EQUAL, 'PLUS_EQUAL'],
+              [Blockly.Msg.MATH_ASSIGNMENT_BY_MINUS_EQUAL, 'MINUS_EQUAL'],
+              [Blockly.Msg.MATH_ASSIGNMENT_BY_TIMES_EQUAL, 'TIMES_EQUAL'],
+              [Blockly.Msg.MATH_ASSIGNMENT_BY_DIVIDE_EQUAL, 'DIVIDE_EQUAL'],
+              [Blockly.Msg.MATH_ASSIGNMENT_BY_MODULO_EQUAL, 'MODULO_EQUAL']];
+        this.setColour(350);
+        this.setPreviousStatement(true, ["STATEMENT", "SET"]);
+        this.setNextStatement(true, ["STATEMENT", "SET"]);
```

Comment:
```suggestion
        this.setPreviousStatement(true, ["STATEMENT"]);
        this.setNextStatement(true, ["STATEMENT"]);
```
STATEMENT is the wildcard connection.